### PR TITLE
fix: Adjust URLs on Video Tab Overview Page

### LINKED ISSUE #6056: Virtual Events: Adjust URLs on Video Tab Overview Page
The video overview page has two columns with links. Please change these columns as follows:

1. Video Source URL: Unlinkify the URL. Reason: We do not really want to direct the user (organizer) to another website or easily enable them to share the source URL by copying the link. We just want to inform them here which service they are using. If they really want to get the entire link they can visit the edit page of the stream.

2. Join Video URL
 * Place the link in a box, but do not make the box and button area use more space than the current implementation. Reason: It is clearer that the "Copy" button is related to the URL and that this is a unit if the URL is boxed. Furthermore the user can inspect the complete URL by moving back and forth in the box. If the user has to copy the URL first and enter it elsewhere it brings the attention away. Showcasing that we integrate video systems nicely in our system by visualizing the URL is a feature that we can show off to expert users. For normal users it does not disturb them neither as at the same time the URL does not take up too much space, because it is boxed.
 * Do not make the URL blue anymore to reduce focus on the link.
 * Change the color of the Copy button from black to gray to not have it stand out so much
 * Behind the current "Copy" button add an arrow button as below that directs the user to the link

Arrow button
Screenshot from 2020-12-17 04-00-15

Screenshot from 2020-12-16 12-51-40

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #6056 

#### Changes proposed in this pull request:

  - [x] Place the link in a box, but do not make the box and button area use more space than the current implementation. Reason: It is clearer that the "Copy" button is related to the URL and that this is a unit if the URL is boxed. Furthermore the user can inspect the complete URL by moving back and forth in the box. If the user has to copy the URL first and enter it elsewhere it brings the attention away. Showcasing that we integrate video systems nicely in our system by visualizing the URL is a feature that we can show off to expert users. For normal users it does not disturb them neither as at the same time the URL does not take up too much space, because it is boxed.
  - [x] Do not make the URL blue anymore to reduce focus on the link.
 - [x] Change the color of the Copy button from black to gray to not have it stand out so much
  - [x]  Behind the current "Copy" button add an arrow button as below that directs the user to the link

#### ScreenShot
Screenshot from 2020-12-21 04-26-27

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by daretobedifferent18** [comment]
@mariobehling Pls could you explain me in detail 1 and 4 point of keeping url in box and implement arrow button behind the copy button.

Implementing arrow button behind the copy button will not show arrow button, couold you elaborate it.
Also making the url inside the box, should I also make them the links linkify after putting them inside the box.

**Comment by iamareebjamal** [comment]
Add arrow button to the right of copy button

**Comment by mariobehling** [comment]
> Also making the url inside the box, should I also make them the links linkify after putting them inside the box.

I think no need to linkify them. After the changes we can see how this works for a few days. Probably no more changes are needed after that.

**Comment by daretobedifferent18** [comment]
@iamareebjamal I am not getting arrow button with simantic icons

**Comment by Afzal-Ind** [comment]
>I am not getting arrow button with simantic icons so I have used other icon. Pls have a look

Plz use share button as arrow button in semantic ui icon. Thanks.

**Comment by daretobedifferent18** [comment]
@iamareebjamal Done with this PR

**Comment by iamareebjamal** [comment]
Field, copy button and link button are unaligned
image

See this for comparison 
image

**Comment by daretobedifferent18** [comment]
@iamareebjamal Done

**Comment by iamareebjamal** [comment]
Still not aligned

**Comment by daretobedifferent18** [comment]
> Still not aligned
> image

aligned as in should I remove the space between two  buttons?
bcz dimensions of bith buttons are now same

**Comment by iamareebjamal** [comment]
See the bottom line of the field and the copy button and compare

image

image

**Comment by maze-runnar** [comment]
it's overlapping with the room password. Please also fix that.
Screenshot from 2020-12-19 09-54-26

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls have a look

**Comment by daretobedifferent18** [comment]
By mistake

**Comment by iamareebjamal** [comment]
Screenshot from 2020-12-21 06-32-30 (1)

It's not taking up the width of the column

**Comment by daretobedifferent18** [comment]
> Screenshot from 2020-12-21 06-32-30 (1)
> 
> It's not taking up the width of the column

Coul you suggest me the way out to deal with it bcz otherwise if I do then the table resp styling will be broken

**Comment by iamareebjamal** [comment]
width: 100% on ui input

Also, instead of making icons 38 px, make ui input height 36px

**Comment by daretobedifferent18** [comment]
Screenshot from 2020-12-21 08-07-48

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls see the changes

**Comment by iamareebjamal** [comment]
Dunno why but video source URL stays in its column but Join Video doesn't

image

This is not true for dev branch 
image

**Comment by daretobedifferent18** [comment]
> Dunno why but video source URL stays in its column but Join Video doesn't
> 
> image
> 
> This is not true for dev branch
> image

Exactly I don't know why this bug is getting. I checked into it but couldn't find anything relevent

**Comment by iamareebjamal** [comment]
Move to other issues for now

**Comment by mariobehling** [comment]
Updated branch, so I can test with latest changes.

File: app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-url.hbs [hunk 1]
```diff
@@ -1,10 +1,18 @@
 {{#if @record}}
-  <div class="d-flex items-center">
-    <a class="truncate" href={{this.link}} target="_blank" rel="noopener">{{this.streamUrl}}</a>
-    <UiPopup @on="click" @content={{t "Link copied to clipboard"}} @position="bottom right" class="ml-auto">
-      <CopyButton @clipboardText={{this.streamUrl}} @class="ui black icon button">
+  <div class="ui action input" style="width: 100%; height: 36px">
+    <input class="truncate" type="text" value={{this.streamUrl}} placeholder={{this.streamUrl}} readonly>
```
Comment (sachinchauhan2889) [inline app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-url.hbs:3]:
```suggestion
    <input class="truncate" style="flex: 1 0 !important" type="text" value={{this.streamUrl}} placeholder={{this.streamUrl}} readonly>
```

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
@iamareebjamal sir, finally solved. I suggest changes. these changes solve issue. Please commit this suggested change in this PR. It takes huge time :yawning_face: .

**Pull request merged**